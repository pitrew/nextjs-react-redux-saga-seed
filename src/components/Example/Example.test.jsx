import React from 'react';
import renderer from 'react-test-renderer';
import { Example } from './Example';

describe('Exmaple', () => {
    test('Renders correctly', () => {
        const jsx = (<Example />);
        const tree = renderer
            .create(jsx)
            .toJSON();

        expect(tree).toMatchSnapshot();
    });
});
