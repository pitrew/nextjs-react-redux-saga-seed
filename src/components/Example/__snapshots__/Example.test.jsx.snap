// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exmaple Renders correctly 1`] = `
<div>
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": Object {},
      }
    }
  />
  <div
    className="example"
  >
    Example
  </div>
</div>
`;
